packages/dids/src/dht.ts

```diff
@@ -0,0 +1,339 @@
+import {Jose, PublicKeyJwk, Web5Crypto} from '@web5/crypto';
```

Comment:
I think `PublicKeyJwk` and `Web5Crypto` are types which we can import explicitly as types.

Reply:
cool didn't know about type imports, what's it do?